src/models/serialization.jl

```diff
@@ -0,0 +1,92 @@
+
+"""Serializes a PowerSystemType to a JSON file."""
+function to_json(obj::T, filename::String) where {T <: PowerSystemType}
+    return open(filename, "w") do io
+        return to_json(io, obj)
+    end
+
+    @info "Serialized $T to $filename"
```

Comment:
Shouldn't this be `@debug`?

Reply:
I intended to always log it.  It won't happen very often.

Reply:
I'm leaving it here unless you still disagree.